Cargo.toml

```diff
@@ -254,3 +255,4 @@ parquet2 = { git = "https://github.com/jorgecarleitao/parquet2", rev = "b0e6545"
 metrics = { git = "https://github.com/datafuse-extras/metrics.git", rev = "fc2ecd1" }
 icelake = { git = "https://github.com/icelake-io/icelake", rev = "f06cdf3" }
 sentry = { git = "https://github.com/getsentry/sentry-rust", rev = "6ef6d97" }
+micromarshal = { git = "https://github.com/ariesdevil/opensrv", rev = "6c96813" }
```

Comment:
Why not give the change back and use https://github.com/datafuselabs/opensrv?

Reply:
micromarshal also need patched ethnum https://github.com/ariesdevil/opensrv/blob/borsh/components/micromarshal/Cargo.toml#L20 
after ethnum upstream merge, I'll modify this.